### PR TITLE
Added hall-of-fame to README

### PR DESCRIPTION
**Pull request by sergey48k**
Hey guys, this is how Hall-of-fame will look like on your repo.  I spoke to @dmpetrov about this PR before making it.

Note: it currently uses my GitHub user (sergey48k) for GitHub API. This takes ~10-20 requests off my limit. Not a biggie, but in case you are wondering why my username is in your source code, it's the reason why. I could also switch to any of you, but the only qualified DVC contributor is @efiop because only he has Sourcerer account linked with a GitHub account.

Also, note. If you accept this pull request, I will become a 'new' contributor to DVC, so my face will likely be on one of the badges for a few days. If you don't want it (and I won't be offended, don't worry), don't accept this request, but just copy the changes, and make a new pull request.

I hope you like it.
Best,
Sergey

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -129,6 +129,30 @@ Contributing
 Contributions are welcome! Please see our `Contributing Guide <https://dvc.org/doc/user-guide/contributing/>`_ for more
 details.
 
+.. image:: https://sourcerer.io/fame/sergey48k/dataversioncontrol/dvc/images/0
```
Comment (shcheklein) [inline README.rst]: should it be iterative/dvc everywhere?

File: README.rst [hunk 2]
```diff
@@ -129,6 +129,30 @@ Contributing
...
+  :target: https://sourcerer.io/fame/sergey48k/dataversioncontrol/dvc/links/0
+
+.. image:: https://sourcerer.io/fame/sergey48k/dataversioncontrol/dvc/images/1
```
Comment (shcheklein) [inline README.rst]: we can def do efiop instead of sergey48k

**Comment by shcheklein** [comment]
I love it! Cool stuff @sergey48k :) 👍

**Comment by sergey48k** [comment]
Addressed @shcheklein's comments.

**Pull request merged**